test/dshell/sameenv.d

```diff
@@ -8,8 +8,8 @@ void main()
     run("$OUTPUT_BASE/printenv$EXE", File(envFromExe, "wb"));
     run("$DMD -m$MODEL -run $EXTRA_FILES/printenv.d", File(envFromRun, "wb"));
 
-    const fromExe = readText(envFromExe);
-    const fromRun = readText(envFromRun);
+    const fromExe = readOutput(envFromExe);
+    const fromRun = readOutput(envFromRun);
```

Comment:
@marler8997 This test from https://github.com/dlang/dmd/pull/8121 didn't work with a debug compiler.

Reply:
Did you fix this issue?  I'm not seeing any failures with this test in the CIs

Reply:
The filtering in `readOutput` is necessary to remove the "DMD 2.087.1-git-1234 DEBUG" message generated by a debug build of dmd. All existing CIs use a release build (i.e. none of the asserts in the compiler is tested), but this PR changes that.

IIRC you were the main contributor of the dshell-tests, so I just wanted to note that these tests should usually use `readOutput` (or something similar) to process compiler output.

Reply:
BTW is printing this debug string on every execution really worth it?
It seems to produce more trouble than gain.

Reply:
With my other suggestion, this becomes readText(envFromRun).filterCompilerDebugMsg

Reply:
> BTW is printing this debug string on every execution really worth it?
> It seems to produce more trouble than gain.

I agree, it's quite a nuisance with little benefit.